java/cloud-function-hello-world/README.md

```diff
@@ -0,0 +1,80 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
+
+# Cloud Functions Hello World with Cloud Code
+
+"Java: Hello World" is a simple HTTP-triggered Cloud Functions application that contains a sample Java-based script that outputs a sample "Hello World" string.
+
+## Table of Contents
+
+* [Directory contents](#directory-contents)
+* [Getting started with VS Code](#getting-started-with-vs-code)
+* [Sign up for user research](#sign-up-for-user-research)
+
+## Directory contents
+* `launch.json` - the required Cloud Code configurations
+* `HelloWorld.java` - the Java "Hello World" sample’s code
+
+## Getting started with VS Code
+
+### Before you begin
+
+1. If you're new to Google Cloud, [create an account](https://console.cloud.google.com/freetrial/signup/tos) to evaluate how our products perform in real-world scenarios. New customers also get $300 in free credits to run, test, and deploy workloads.
+
+1. In the Google Cloud console, on the project selector page, select or [create a Google Cloud project](https://console.cloud.google.com/projectselector2/home/dashboard).
+
+    **Note**: If you don't plan to keep the resources that you create in this procedure, create a project instead of selecting an existing project. After you finish these steps, you can delete the project, removing all resources associated with the project.
+
```

Comment:
*Nit* Apply the same framing from the earlier suggestion here.

Reply:
Same as before, item raised; thanks!